openapi-core/src/main/java/io/ballerina/openapi/core/generators/service/model/OASServiceMetadata.java

```diff
@@ -67,8 +67,8 @@ public static class Builder {
         private Filter filters;
         private boolean nullable = false;
 
-        private String license = "// AUTO-GENERATED FILE. DO NOT MODIFY.\n\n" +
-                "// This file is auto-generated by the Ballerina OpenAPI tool.\n";
+        private String license = "// AUTO-GENERATED FILE. \n" +
+                "// This file is auto-generated by the Ballerina OpenAPI tool.\n\n";
```

Comment:
Can't we keep a common constant for this instead of keep the same file header in multiple places?

Reply:
Move to common constant file